### PR TITLE
[AC-1266] Enums filename conventions

### PR DESCRIPTION
**Pull request by vincentsalucci**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Before you submit
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by eliykat — COMMENTED** [review]
I think the filename format would be (for example) `kdf-type.enum.ts`. In keeping with `.component.ts`, `.service.ts`, etc.

Example: https://github.com/bitwarden/clients/blob/master/libs/components/src/dialog/simple-configurable-dialog/models/simple-dialog-type.enum.ts

File: libs/common/src/vault/models/view/card.view.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Jsonify } from "type-fest";
 
-import { CardLinkedId as LinkedId } from "../../../enums/linkedIdType";
+import { CardLinkedId as LinkedId } from "../../../enums/linked-id-type";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply libs/common/src/vault/models/view/card.view.ts]: Fixed! Thank you.

File: libs/common/src/vault/models/view/login.view.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Jsonify } from "type-fest";
 
-import { LoginLinkedId as LinkedId } from "../../../enums/linkedIdType";
+import { LoginLinkedId as LinkedId } from "../../../enums/linked-id-type";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply libs/common/src/vault/models/view/login.view.ts]: Fixed! Thank you.

File: apps/desktop/native-messaging-test-runner/src/commands/bw-credential-create.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import "module-alias/register";
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 
-import { NativeMessagingVersion } from "@bitwarden/common/enums/nativeMessagingVersion";
+import { NativeMessagingVersion } from "@bitwarden/common/enumsmessaging-version";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply apps/desktop/native-messaging-test-runner/src/commands/bw-credential-create.ts]: Fixed! Thank you.

File: apps/desktop/native-messaging-test-runner/src/commands/bw-credential-retrieval.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import "module-alias/register";
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 
-import { NativeMessagingVersion } from "@bitwarden/common/enums/nativeMessagingVersion";
+import { NativeMessagingVersion } from "@bitwarden/common/enumsmessaging-version";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply apps/desktop/native-messaging-test-runner/src/commands/bw-credential-retrieval.ts]: Fixed! Thank you.

File: apps/desktop/native-messaging-test-runner/src/commands/bw-credential-update.ts [hunk 5]
```diff
@@ -3,7 +3,7 @@ import "module-alias/register";
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 
-import { NativeMessagingVersion } from "@bitwarden/common/enums/nativeMessagingVersion";
+import { NativeMessagingVersion } from "@bitwarden/common/enumsmessaging-version";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply apps/desktop/native-messaging-test-runner/src/commands/bw-credential-update.ts]: Fixed! Thank you.

File: apps/desktop/native-messaging-test-runner/src/commands/bw-generate-password.ts [hunk 6]
```diff
@@ -3,7 +3,7 @@ import "module-alias/register";
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 
-import { NativeMessagingVersion } from "@bitwarden/common/enums/nativeMessagingVersion";
+import { NativeMessagingVersion } from "@bitwarden/common/enumsmessaging-version";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply apps/desktop/native-messaging-test-runner/src/commands/bw-generate-password.ts]: Fixed! Thank you.

File: apps/desktop/native-messaging-test-runner/src/commands/bw-handshake.ts [hunk 7]
```diff
@@ -1,6 +1,6 @@
 import "module-alias/register";
 
-import { NativeMessagingVersion } from "@bitwarden/common/enums/nativeMessagingVersion";
+import { NativeMessagingVersion } from "@bitwarden/common/enumsmessaging-version";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply apps/desktop/native-messaging-test-runner/src/commands/bw-handshake.ts]: Fixed! Thank you.

File: apps/desktop/native-messaging-test-runner/src/commands/bw-status.ts [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import "module-alias/register";
 
-import { NativeMessagingVersion } from "@bitwarden/common/enums/nativeMessagingVersion";
+import { NativeMessagingVersion } from "@bitwarden/common/enumsmessaging-version";
```
[the remark this answers is not in the document]
Comment (vincentsalucci) [reply apps/desktop/native-messaging-test-runner/src/commands/bw-status.ts]: Fixed! Thank you.

**Review by eliykat — APPROVED** [review]
✅ enum files match filename conventions
✅ all imports use the barrel file

Nice work 🧹

**Pull request merged**